products/redis/api.yaml

```diff
@@ -44,8 +44,7 @@ async: !ruby/object:Api::Async
 objects:
   - !ruby/object:Api::Resource
     name: 'Instance'
-    base_url: |
```

Comment:
This is effectively still a no-op; using a block scalar (`|`) indicates that you're using a multiline string *including newlines* which is why these needed to be changed.

Before, we coerced paragraphs into single strings. Now, we're more restrictive and only accept true strings.